### ISSUE TITLE
Pandora has stopped working

### ISSUE THREAD

**Issue by kevindb** [msfeldstein/chrome-media-keys#120]
Media Keys recently stopped working with pandora.com. It simply always reports "There's nothing playing right now" (see attached screenshot). The timeline seems like it would fit the June 20 release of 4.60.
If there's anything else I can do to help, such as providing logs or testing, just let me know.
Thank you for a wonderful extension!

![screen shot 2017-06-26 at 7 57 59 am](https://user-images.githubusercontent.com/3468013/27538388-b605a43a-5a45-11e7-8578-b0918de60b20.png)

**Comment by RadValentin** [comment]
@kevindb The way the plugin works is by looking for page elements with certain CSS classes and clicks them to play/pause/etc. Can you check that the classes defined [here](https://github.com/msfeldstein/chrome-media-keys/blob/master/src/controllers/PandoraController.js) match the playback buttons in Pandora.

**Comment by kevindb** [comment]
The class names in the second section (starting on [line 25](https://github.com/msfeldstein/chrome-media-keys/blob/62b0b17/src/controllers/PandoraController.js#L25)) are correct. However, I suspect the [isPlaying](https://github.com/msfeldstein/chrome-media-keys/blob/62b0b17/src/controllers/PandoraController.js#L45) section is outdated.
If I understand your code, [Line 49](https://github.com/msfeldstein/chrome-media-keys/blob/62b0b17/src/controllers/PandoraController.js#L49) is looking for a string "#ic_pause". When I inspect the svg, the xlink:href does not contain the string "#ic_pause".

This is what it looks like when music _IS_ playing:
<img width="512" alt="screen shot playing" src="https://user-images.githubusercontent.com/3468013/28077079-afa9925a-662e-11e7-81cd-473cdef12d68.png">

This is when the music IS _NOT_ playing:
<img width="512" alt="screen shot paused" src="https://user-images.githubusercontent.com/3468013/28077205-1ab95ec2-662f-11e7-8b5c-8e025d212ac7.png">

Could this be the cause of the issue?

**Comment by RadValentin** [comment]
Yep, most likely, you'd need to check the aria attribute `aria-labelledby` to determine the play/pause state.

**Comment by kevindb** [comment]
If you wanted to continue using the attribute `xlink:href`, you could look for the string `#controls_pause` or `#controls_play` instead of the current `#ic_pause`.

**Comment by kevindb** [comment]
I just that extension v4.61 is working with Pandora. Thank you!

**Comment by kevindb** [comment]
I spoke too soon. The extension has _not_ been working with Pandora for the last two days.

**Comment by kevindb** [comment]
Could we reopen this issue, please? Pandora has stopped working, with the same behavior as I originally described.
